build-container/entrypoint.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+set -e
+
+if [ ! -z "${GPG_KEY_DATA-}" ]; then
+    if [ -z "${GPG_KEY_ID-}" ]; then
+        echo 'GPG_KEY_ID is not set, but GPG_KEY_DATA is set. Please set GPG_KEY_ID to the key ID of the key.'
+        exit 1
+    fi
+    echo "${GPG_KEY_DATA}" | gpg --import /dev/stdin
```

Comment:
A slightly safer way to do it, which wouldn't expose the key even with `set -x`, could be
```sh
gpg --import <<<"${GPG_KEY_DATA}"
```

Reply:
Oh, I completely forgot about the `<<<`. Yeah that'd be neater for sure